catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/DeleteOperations.java

```diff
@@ -168,7 +164,14 @@ public DeleteResponse doDelete(DeleteRequest deleteRequest, List<String> fanoutT
                             .size());
 
             deleteResponse = performLocalDelete(deleteRequest, deleteStorageRequest);
-            deleteResponse = performRemoteDelete(deleteRequest, deleteResponse);
+
+            RemoteDeleteOperations remoteDeleteOperations = new RemoteDeleteOperations(
```

Comment:
Why are we `new`ing this for every operation instead of making it a bean and injecting it?